### PR TITLE
Improve React ES6

### PR DESCRIPTION
**Pull request by mauroporrasp**
These are then changes that I included in https://github.com/honza/vim-snippets/pull/799

### CODE REVIEW

File: snippets/javascript.es6.react.snippets [hunk 1]
```diff
@@ -22,17 +24,19 @@ snippet rcon
 		super(props)
 
 		this.state = {
-			${1}: ${0},
+			${1:foo}: ${0:'foo'},
```
Comment (lpil) [inline snippets/javascript.es6.react.snippets]: No placeholders please :)

File: snippets/javascript.es6.react.snippets [hunk 2]
```diff
@@ -41,12 +45,12 @@ snippet rcdm
 	}
 
 # State
-snippet rsst
+snippet rtss
 	this.setState({
-		${1}: ${0},
+		${1:foo}: ${0:'foo'},
```
Comment (lpil) [inline snippets/javascript.es6.react.snippets]: Here too

**Comment by mauroporrasp** [comment]
@lpil I'm using the "reduced" syntactic sugar style (without semicolons) for ECMAScript 6
https://github.com/rse/es6-features#frequently-asked-questions-faq

**Comment by lpil** [comment]
Aye, but this repo is using the traditional semicolon style :)

**Comment by mauroporrasp** [comment]
@lpil What would be the best way to use the reduced version? Keep a fork?

**Comment by lpil** [comment]
Possibly, unless there's a clever way to have snippets with configurable semicolons

**Pull request closed without merging**